dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A database browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite. \
+It is for users and developers wanting to create databases, search, and edit data. \
+It uses a familiar spreadsheet-like interface, and you don't need to learn complicated\ SQL commands.\
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="COPYRIGHT 2012-2014 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+SOURCE_FILENAME="sqlitebrowser-$portVersion.tar.gz"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
+    "
+REQUIRES="
+        haiku >= $haikuVersion
+        "
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsqlite3
+	devel:libqt5
+         devel:libphonon
+	devel:libQt3Support
+	devel:libQtCLucene
+	devel:libQtCore
+	devel:libQtDBus
+	devel:libQtDeclarative
+	devel:libQtDesigner
+	devel:libQtDesignerComponents
+	devel:libQtGui
+	devel:libQtHelp
+	devel:libQtMultimedia
+	devel:libQtNetwork
+	devel:libQtScript
+	devel:libQtScriptTools
+	devel:libQtSql
+	devel:libQtSvg
+	devel:libQtTest
+	devel:libQtWebKit
+	devel:libQtXml
+	devel:libQtXmlPatterns
+	devel:libQtUiTools
+         	"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:assistant
+	cmd:designer
+	cmd:lconvert
+	cmd:linguist
+	cmd:lrelease
+	cmd:lupdate
+	cmd:moc
+	cmd:pixeltool
+	cmd:qcollectiongenerator
+	cmd:qdbuscpp2xml
+	cmd:qdbusviewer
+	cmd:qdbusxml2cpp
+	cmd:qdoc3
+	cmd:qhelpconverter
+	cmd:qhelpgenerator
+	cmd:qmake
+	cmd:qmlplugindump
+	cmd:qmlviewer
+	cmd:qt3to4
+	cmd:qttracereplay
+	cmd:rcc
+	cmd:uic
+	cmd:uic3
+	cmd:xmlpatterns
+	cmd:xmlpatternsvalidator
```

Comment:
Yep, my approach was a bit excessive ;-)
@korli's method is much better indeed.